packages/web-react/src/components/Item/demo/index.tsx

```diff
@@ -1,3 +1,5 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
```

Comment:
Will it work with the same syntax as on other lines?

```suggestion
// eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved
```

Reply:
Fixed.